allium/lib/page_writer.py

```
@@ -744,6 +744,68 @@ def get_directory_authorities_data(relay_set):
+    #
+    # Display-only: these entries are never counted as voters and never affect the
+    # reachability/consensus denominators. Totals below use the DYNAMIC (pre-merge)
+    # authority count so the hardcoded rows can't inflate any counts.
+    # ------------------------------------------------------------------
+    dynamic_authority_count = len(authorities)
+    dynamic_fingerprints = {a.get('fingerprint', '').upper() for a in authorities}
+    dynamic_nicknames = {a.get('nickname', '').lower() for a in authorities}
+    try:
+        from .consensus.collector_fetcher import get_known_offline_authorities
+        known_offline = get_known_offline_authorities()
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)